ui/src/shared/components/BoxTooltip.scss

```diff
@@ -4,20 +4,89 @@ $box-tooltip--caret-size: 6px;
   position: fixed;
   visibility: hidden;
   z-index: $z--dygraph-legend;
-  border: $ix-border solid $c-pool;
+}
+
+.box-tooltip--contents {
+  border-width: $ix-border;
+  border-style: solid;
   background-color: $g1-raven;
   border-radius: $radius;
   padding: 10px;
   font-size: 13px;
+  word-break: break-all;
+  word-wrap: break-word;
+  width: calc(100% - #{$box-tooltip--caret-size * 2});
+
+  pre {
```

Comment:
I viewed the box tooltip as a generic container component that you could put anything inside. It seems like these additional `.box-tooltip--contents` styles belong on the child component. What do you think? Want to keep the box tooltip / composable.